### ISSUE TITLE
Cannot log in http://khon.space - process returns page unavailable

### ISSUE THREAD

**Issue by revenantal** [Kyria/LazyBlacksmith#65]
Attempting to log in with any accounts (I am a new user, account previously not registered) at https://khon.space/ successfully redirects to EVE SSO, upon doing the redirect back to khon.space however the page returns.

![image](https://user-images.githubusercontent.com/5674404/51147215-6fcce180-1827-11e9-90d5-d4047d638664.png)

No Additional details provided.

**Comment by Kyria** [comment]
I'll try to resolve this as quickly as possible, as it looks like I have some issues with the esi/verify endpoint (as khon.space is still using master version, without SSOv2). 
Time to finish my py3 move and SSOv2...

**Comment by revenantal** [comment]
Gotcha, I messaged you on slack. Give me a shout if you need a hand testing at all.

**Comment by Kyria** [comment]
This should be fixed with the last deployement i've made. Thanks for reporting the issue !

**Comment by inkogandersnito** [comment]
I ran into the same error message when logging on to https://lazy-blacksmith.space with Account-1 today. I tried two other accounts, Account-2 and Account-3, and they both work fine.

The blueprint list loads without problem. It is when I click on manufacturing, ie https://lazy-blacksmith.space/blueprint/manufacturing/29040/10/20, that I get the error message as per the first post in this thread.
